aria/orchestrator/workflows/builtin/utils.py

```diff
@@ -68,3 +67,11 @@ def get_task(node_instance_id):
                     graph.add_dependency(dependency, task)
             else:
                 graph.add_dependency(task, dependencies)
+
+
+def _has_operation(interfaces, operation_name):
+    for interface in interfaces:
+        for operation in interface.operations:
```

Comment:
yup, thats an sqla syntax. there is a `exists` concept in sqlalcehmy, but from what i've seen it includes importing sqla into the module...
anyway, there's this example:
(ret, ), = Session.query(exists().where(Keyword.name.in_(keywords)))